### PR TITLE
B709

### PR DESCRIPTION
**Pull request by melkonyan**

### CODE REVIEW

**Comment by melkonyan** [comment]
This should fix #709

File: dexlib2/src/main/java/org/jf/dexlib2/util/Preconditions.java [hunk 1]
```diff
@@ -221,7 +221,7 @@ public static int checkArrayPayloadElementWidth(int elementWidth) {
 
     public static <L extends List<? extends Number>> L checkArrayPayloadElements(int elementWidth, L elements) {
         // mask of all bits that do not fit into an 'elementWidth'-bit number
```
Comment (aki-ks) [inline dexlib2/src/main/java/org/jf/dexlib2/util/Preconditions.java]: It should be `an 'elementWidth'-byte number`

Comment (aki-ks) [inline dexlib2/src/main/java/org/jf/dexlib2/util/Preconditions.java]: It should be `an 'elementWidth'-byte number`.

Comment (melkonyan) [reply dexlib2/src/main/java/org/jf/dexlib2/util/Preconditions.java]: Done

**Comment by melkonyan** [comment]
Friendly ping:)

**Comment by JesusFreke** [comment]
Merged as b044a0035391f4d1accddae1a78bf91b42b03b0c

**Comment by JesusFreke** [comment]
Thanks for the patch!

**Comment by aki-ks** [comment]
@JesusFreke Have you noticed my latest comment in #705? I've not noticed that my IDE filled in zeros instead of field accesses.

**Comment by JesusFreke** [comment]
Were you planning on sending a PR? If not, I'll go ahead and fix it.

**Comment by aki-ks** [comment]
You can just fix it, that's way faster.

**Pull request closed without merging**